lib/engines/content_object_store/features/edit_object.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Edit a content object
```

Comment:
After a convo with @pezholio I haven't continued extending our integration here as it should be covered by this Cucumber test. Between this and the unit tests for the service it feels well covered. If I were to add more to the integration test I'd be repeating a lot of what's written in the units?

Reply:
are these more 'expensive' than the integration tests do you think?
just wondering what we should do going forward...should features be for one main happy path and just test what's expected in the UI, and then integration is for happy and unhappy, and testing the underlying database?
e.g. you could put validation/error handling in integration

Reply:
I should think Cucumber will be more expensive to run.

Before reading this I went and followed what the Create journey did and put Updates tests in Cucumber too! These did feel like common and expected scenarios from a users point of view, so felt worthy of the cost?

Defining what and when to cover code with integration tests would be good. 

At the moment we've reasonably moved the majority of the logic out of controllers and into service classes. Those services are stubbing and testing the integrations to Publishing API. 

I think a good example for integration will be something like scheduled publishing or notifications. We wouldn't have cucumber tests but would have integration and units?

Cucumber: user facing happy paths
Integration: edge cases or not user facing paths